pkg/daemon/rpm-ostree.go

```diff
@@ -245,17 +297,82 @@ func (r *RpmOstreeClient) Rebase(imgURL, osImageContentDir string) (changed bool
 	args := []string{"rebase", "--experimental", fmt.Sprintf("%s:%s", repo, ostreeCsum),
 		"--custom-origin-url", customURL, "--custom-origin-description", "Managed by machine-config-operator"}
 
-	var out []byte
-	if out, err = runGetOut("rpm-ostree", args...); err != nil {
-		// capture stdout output as well in case of error
-		err = errors.Wrapf(err, "with stdout output: %v", string(out))
-		return
-	}
-
+	_, err = runRpmOstree(args[0], args[1:]...)
 	changed = true
 	return
 }
 
+// constant for describing kernel argument operations
+const (
+	kargRemove int = iota
+	kargAdd
+)
+
+// KernelArgument describes an operation for karg handling
+type KernelArgument struct {
+	Operation int
+	Name      string
+}
+
+// runRpmOStree wraps the rpm-ostree command. Unless mocked, this
+// is the rpmOstreeCommander that is used by RpmOstreeClient.
+func runRpmOstree(noun string, args ...string) ([]byte, error) {
+	if len(args) > 2 {
+		return nil, fmt.Errorf("rpm-ostree command is malformed, not enough arguments")
+	}
+	glog.Infof("Executing cmd: '%s %s'", realRpmOstreeCmd, args)
+	out, err := runGetOut(realRpmOstreeCmd, args...)
+	if err != nil {
+		glog.Errorf("'%s %v' failed to run: %v:\n %s", realRpmOstreeCmd, args, string(out), err)
+	}
+	return out, err
+}
+
+// SetKernelArgs sets kernel arguments on an RPM OStree systeR
```

Comment:
`RPM OStree systeR` -> `rpm-ostree system`? Or was this referring to something else